src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_tue calendar--month-length_30">
+      <div class="calendar__cell calendar__cell--1"></div>
+      <div class="calendar__cell calendar__cell--2"></div>
+      <div class="calendar__cell calendar__cell--3"></div>
+      <div class="calendar__cell calendar__cell--4"></div>
+      <div class="calendar__cell calendar__cell--5"></div>
+      <div class="calendar__cell calendar__cell--6"></div>
+      <div class="calendar__cell calendar__cell--7"></div>
+      <div class="calendar__cell calendar__cell--8"></div>
+      <div class="calendar__cell calendar__cell--9"></div>
+      <div class="calendar__cell calendar__cell--10"></div>
+      <div class="calendar__cell calendar__cell--11"></div>
+      <div class="calendar__cell calendar__cell--12"></div>
+      <div class="calendar__cell calendar__cell--13"></div>
+      <div class="calendar__cell calendar__cell--14"></div>
+      <div class="calendar__cell calendar__cell--15"></div>
+      <div class="calendar__cell calendar__cell--16"></div>
+      <div class="calendar__cell calendar__cell--17"></div>
+      <div class="calendar__cell calendar__cell--18"></div>
+      <div class="calendar__cell calendar__cell--19"></div>
+      <div class="calendar__cell calendar__cell--20"></div>
+      <div class="calendar__cell calendar__cell--21"></div>
+      <div class="calendar__cell calendar__cell--22"></div>
+      <div class="calendar__cell calendar__cell--23"></div>
+      <div class="calendar__cell calendar__cell--24"></div>
+      <div class="calendar__cell calendar__cell--25"></div>
+      <div class="calendar__cell calendar__cell--26"></div>
+      <div class="calendar__cell calendar__cell--27"></div>
+      <div class="calendar__cell calendar__cell--28"></div>
+      <div class="calendar__cell calendar__cell--29"></div>
+      <div class="calendar__cell calendar__cell--30"></div>
+      <div class="calendar__cell calendar__cell--31"></div>
```

Comment:
Create a markup for the calendar block with 31 days inside
 * DON'T add numbers in HTML (you will do it using CSS)